gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/AddOrEditGradeItemPanelContent.html

```diff
@@ -45,11 +45,11 @@
 					<sakai-rubric-association wicket:id="sakai-rubric-association"></sakai-rubric-association>
 				</div>
 			</div>
-			<div class="input-group input-group-sm mb-3">
+			<div class="d-flex align-items-center input-group input-group-sm">
 				<label wicket:for="duedate" class="col-xs-3 control-label">
 					<wicket:message key="label.addgradeitem.due" />
 				</label>
-				<div class="d-flex align-items-center justify-content-evenly gb-input-div">
+				<div class="d-flex align-items-center justify-content-evenly ms-3">
 					<input wicket:id="duedate" type="text" class="form-control gb-datepicker gb-input" aria-describedby="duedate-help" />
```

Comment:
can you make due date look like the other inputs (E.g., on the next line instead of inline)?

Reply:
i have some more gradebook jira left to do, i will incorporate this in one of those jiras. i agree it would look better.

Reply:
I'll prefer the category inline. This information is on a modal. Try avoid a lot of scroll

Reply:
make sense. the modal shouldn't need to be scrolled to create an gb item